frontend/src/components/TopicCard/Info.js

```diff
@@ -0,0 +1,104 @@
+/* global $ */
+
+import React, { PropTypes, Component } from 'react'
```

Comment:
Here and any other new components you made, you'll get a console warning unless you remove `PropTypes` from this line and add this line:

     import PropTypes from 'prop-types'

Because I updated react

Reply:
kk, I will go through and update any outstanding